selfdrive/car/chrysler/values.py

```diff
@@ -56,6 +60,11 @@ def __init__(self, CP):
 RAM_HD = {CAR.RAM_HD, }
 RAM_CARS = RAM_DT | RAM_HD
 
+# the increased steer rate hasn't been verified on these cars.
+# remove from this list once it's been tested and confirmed to not fault
+CHRYSLER_OLD_TUNING_BLACKLIST = {CAR.PACIFICA_2017_HYBRID, CAR.PACIFICA_2018, CAR.PACIFICA_2018_HYBRID,
```

Comment:
@nuwandavek the cars in this list will have the old steering rate + PID tuning. you can just remove models from the list and merge when they're ready